pom.xml

```diff
@@ -67,6 +68,7 @@
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
         <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
         <junit-jupiter.version>5.6.2</junit-jupiter.version>
+        <aws-embedded-metrics.version>0.1.0-beta</aws-embedded-metrics.version>
```

Comment:
There is a v1.0.0 now 

https://repo.maven.apache.org/maven2/software/amazon/cloudwatchlogs/aws-embedded-metrics/1.0.0/